packages/cursorless-vscode-e2e/src/suite/performance.vscode.test.ts

```diff
@@ -0,0 +1,140 @@
+import {
+  asyncSafety,
+  type ActionDescriptor,
+  type ScopeType,
+  type SimpleScopeTypeType,
+} from "@cursorless/common";
+import { openNewEditor, runCursorlessCommand } from "@cursorless/vscode-common";
+import assert from "assert";
+import * as vscode from "vscode";
+import { endToEndTestSetup } from "../endToEndTestSetup";
+
+const testData = generateTestData();
+const numLines = testData.split("\n").length;
+
+suite(`Performance: ${numLines} lines JSON`, async function () {
+  endToEndTestSetup(this);
+
+  let previousTitle = "";
+
+  this.beforeEach(function () {
+    const title = this.currentTest!.title;
+    if (title !== previousTitle) {
+      console.log(`    ${title}`);
+      previousTitle = title;
+    }
+  });
+
+  const textBasedThreshold = 100;
+  const parseTreeThreshold = 500;
+  const surroundingPairThreshold = 30000;
```

Comment:
let's put `Ms` on the end of each of these var names to indicate unit

Reply:
done